### ISSUE TITLE
Transcode existing videos

### ISSUE THREAD

**Issue by fflorent** [Chocobozzz/PeerTube#568]
* **PeerTube version or URL:** 1.0.0-beta.4

It sounds like currently there is no way to trigger the transcoding if there are already videos uploaded. Unfortunately I have uploaded bunch of them before activating the transcoding :/.

Florent

**Comment by fflorent** [comment]
I would like to know if a workaround is possible. Here is what I have in mind: 
 - transcode an existing video using ffmpeg manually ;
 - put the file at the right place (data/video directory) ;
 - insert an entry in the `"videoFile"` table with the required information (resolution, size, extname, infoHash, …) ;

However, @Chocobozzz, I remember you told me updating the DB may disturb the fediverse (or something like that). Is it something safe to insert entries in the "videoFile" table or you discourage it?

**Comment by Chocobozzz** [comment]
Yes please don't insert entries in the database, as we do more things in controllers than just inserting some data in PostgreSQL (federation but not only).

If you really want a workaround you could just try to create transcoding jobs manually, but I never did this and I can't guarantee that it will work.

**Comment by fflorent** [comment]
Using Redis ?
